src/transport-http-server.ts

```diff
@@ -102,18 +130,23 @@ export class TransportHttpServer<BagType extends FnsBag> implements ITransport<B
                 return conn;
             }
         }
+
         log('Connection does not exist already.  Making a new one.');
         const conn = new Connection({
             description: `connection to ${otherDeviceId}`,
             transport: this,
             deviceId: this.deviceId,
             methods: this.methods,
-            sendEnvelope: async (conn, env) => {
+            sendEnvelope: (_conn, env) => {
```

Comment:
FYI the `conn` argument only exists here because in some other cases I needed to refer to the connection object from within `sendEnvelope`... while running the Connection constructor, so there was no easy way to get a handle on the connection object except by having it be passed into the function here.  Bit of a hack.